src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/AuthorizationServer/ApiAccess/form_theme.html.twig

```diff
@@ -0,0 +1,19 @@
+{% use '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig' %}
+
+{% block api_access_widget %}
+  <div class="card-body">
+    <div class="form-wrapper">
```

Comment:
I think with this you have duplicated the card layout, since the card is already displayed in `form.html.twig` you don't need to display it again in the widget

Reply:
```suggestion
```